indexer/sql/src/active_model.rs

```diff
@@ -0,0 +1,95 @@
+use {
+    crate::entity,
+    grug_math::Inner,
+    grug_types::{BlockInfo, Tx, TxOutcome},
+    sea_orm::{prelude::*, sqlx::types::chrono::TimeZone, Set},
+};
+
+#[derive(Debug, Default)]
+pub struct Models {
+    pub block: entity::blocks::ActiveModel,
+    pub transactions: Vec<entity::transactions::ActiveModel>,
+    pub messages: Vec<entity::messages::ActiveModel>,
+    pub events: Vec<entity::events::ActiveModel>,
+}
+
+impl Models {
+    pub fn push(&mut self, tx: &Tx, tx_outcome: &TxOutcome) {
```

Comment:
This function can panic. It's called as task with tokio, so only the task should be terminated. Anyway it would be better to return a Result.

Also because this function is called only inside `BlockToIndex::save` function. which already return ` Result<(), sea_orm::DbErr>`

Reply:
Yeah good point.